src/components/form/FieldElement.tsx

```diff
@@ -41,7 +47,7 @@ export default class FieldElement extends React.Component<FieldElementProps, Fie
         this.createNewOption = this.createNewOption.bind(this);
     }
 
-    private onChange([field]: any): void {
+    private onChange(field: SearchResult): void {
```

Comment:
`FieldElement` now expects a single object because `SelectSearch` will use the `onChange` handler